src/common/wrap/wrap-function.js

```diff
@@ -128,6 +140,9 @@ export function createWrapperWithEmitter (emitter, always) {
       const method = methods[i]
       const fn = obj[method]
 
+      if (overwriteContext && fn[flag]) {
+        fn[getContextName(prefix, method)] = getContext
```

Comment:
We may need to be careful here about mutating the original function like this. let's mob as a group about this PR